lib/net.js

```diff
@@ -1496,7 +1496,15 @@ Server.prototype.listen = function(...args) {
     return this;
   }
 
-  throw new ERR_INVALID_OPT_VALUE('options', util.inspect(options));
+  let additionalMessage;
+  if (typeof options === 'object' &&
+    !(('port' in options) || ('path' in options))) {
+
+    additionalMessage = '"port" or "path" must be specified in options.';
+  }
+
+  throw new ERR_INVALID_OPT_VALUE('options', util.inspect(options),
```

Comment:
This error message will not be accurate since `options` is an argument, not an option value - the user does not pass `{ options: { ... } }` to this method. `ERR_INVALID_ARG_VALUE` may be more accurate.